services/blockchain-indexer/shared/indexer/transactionProcessor/dpos/voteDelegate.js

```diff
@@ -26,9 +26,13 @@ const config = require('../../../../config');
 const logger = Logger();
 
 const MYSQL_ENDPOINT = config.endpoints.mysql;
-const votesIndexSchema = require('../../../database/schema/votes');
+const stakesIndexSchema = require('../../../database/schema/stakes');
 
-const getVotesIndex = () => getTableInstance('votes', votesIndexSchema, MYSQL_ENDPOINT);
+const getVotesIndex = () => getTableInstance(
```

Comment:
can we rename to `getVotesTable` instead?

Reply:
This part of the code is out of concern for this issue. It is handled as part of https://github.com/LiskHQ/lisk-service/issues/1347.